### ISSUE TITLE
Balloons not working - Issue on console related to LoneLibs/ItemsAdder?

### ISSUE THREAD

**Issue by reneg1990** [PluginBugs/Issues-CosmeticsCore#127]
### Terms

- [X] I'm using the very latest version of CosmeticsCore and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-CosmeticsCore/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://lonedev.gitbook.io/cosmeticscore/) to know if a solution is already known.
- [X] I already searched on the **Discord** server to know if anyone already provided a solution in the past for the issue.

### Discord tag (optional)

Ren.#1990

### What happened?

Tried to equip a Balloon, didnt appear and theres a Interaction Event on the console misfiring.

### Steps to reproduce the issue

/Wardrobe
Click Balloons
Expected Balloon to be there - no balloon tho, error on console instead.

### Server version

This server is running Paper version git-Paper-307 (MC: 1.19.2) (Implementing API version 1.19.2-R0.1-SNAPSHOT) (Git: 476ef25)
You are running the latest version
Previous version: git-Pufferfish-47 (MC: 1.19.2)

This is Pufferfish+ - It has also been tested on a normal paper 1.19.2 server tho.

### CosmeticsCore Version

CosmeticsCore Version 1.2.2

### ProtocolLib Version

5.0.0-SNAPSHOT-b608

### LoneLibs Version

1.0.2.4

### Error (optional)

```shell
[18.04 21:59:09] [Server] [ERROR] Could not pass event InventoryClickEvent to LoneLibs v1.0.24
[18.04 21:59:09] [Server] java.lang.NullPointerExceptionCannot read field "N" because "this.a" is null
[18.04 21:59:09] [Server] 	at ia.m.aa.a(SourceFile:239) ~[ItemsAdder_3.4.1-r6.jar:?]
[18.04 21:59:09] [Server] 	at ia.m.aa.b(SourceFile:188) ~[ItemsAdder_3.4.1-r6.jar:?]
[18.04 21:59:09] [Server] 	at ia.m.Y.a(SourceFile:470) ~[ItemsAdder_3.4.1-r6.jar:?]
[18.04 21:59:09] [Server] 	at dev.lone.itemsadder.api.CustomEntity.spawn(SourceFile:100) ~[ItemsAdder_3.4.1-r6.jar:?]
[18.04 21:59:09] [Server] 	at dev.lone.cosmeticscore.main.u.Q(SourceFile:58) ~[CosmeticsCore_1.2.2.jar:?]
[18.04 21:59:09] [Server] 	at dev.lone.cosmeticscore.main.T.a(SourceFile:33) ~[CosmeticsCore_1.2.2.jar:?]
[18.04 21:59:09] [Server] 	at dev.lone.cosmeticscore.main.p.a(SourceFile:444) ~[CosmeticsCore_1.2.2.jar:?]
[18.04 21:59:09] [Server] 	at dev.lone.cosmeticscore.main.p.a(SourceFile:325) ~[CosmeticsCore_1.2.2.jar:?]
[18.04 21:59:09] [Server] 	at fr.mrmicky.fastinv.FastInv.handleClick(FastInv.java:316) ~[LoneLibs.jar:?]
[18.04 21:59:09] [Server] 	at fr.mrmicky.fastinv.FastInvManager$1.onInventoryClick(FastInvManager.java:50) ~[LoneLibs.jar:?]
[18.04 21:59:09] [Server] 	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:40) ~[paper-api-1.19.2-R0.1-SNAPSHOT.jar:?]
[18.04 21:59:09] [Server] 	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[paper-api-1.19.2-R0.1-SNAPSHOT.jar:git-Paper-307]
[18.04 21:59:09] [Server] 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.19.2-R0.1-SNAPSHOT.jar:?]
[18.04 21:59:09] [Server] 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:670) ~[paper-api-1.19.2-R0.1-SNAPSHOT.jar:?]
[18.04 21:59:09] [Server] 	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleContainerClick(ServerGamePacketListenerImpl.java:3307) ~[?:?]
[18.04 21:59:09] [Server] 	at net.minecraft.network.protocol.game.ServerboundContainerClickPacket.handle(ServerboundContainerClickPacket.java:58) ~[?:?]
[18.04 21:59:09] [Server] 	at net.minecraft.network.protocol.game.ServerboundContainerClickPacket.handle(ServerboundContainerClickPacket.java:23) ~[?:?]
[18.04 21:59:09] [Server] 	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:51) ~[?:?]
[18.04 21:59:09] [Server] 	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.2.jar:git-Paper-307]
[18.04 21:59:09] [Server] 	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
[18.04 21:59:09] [Server] 	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
[18.04 21:59:09] [Server] 	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1341) ~[paper-1.19.2.jar:git-Paper-307]
[18.04 21:59:09] [Server] 	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:185) ~[paper-1.19.2.jar:git-Paper-307]
[18.04 21:59:09] [Server] 	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
[18.04 21:59:09] [Server] 	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1318) ~[paper-1.19.2.jar:git-Paper-307]
[18.04 21:59:09] [Server] 	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1311) ~[paper-1.19.2.jar:git-Paper-307]
[18.04 21:59:09] [Server] 	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
[18.04 21:59:09] [Server] 	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1289) ~[paper-1.19.2.jar:git-Paper-307]
[18.04 21:59:09] [Server] 	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1177) ~[paper-1.19.2.jar:git-Paper-307]
[18.04 21:59:09] [Server] 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:305) ~[paper-1.19.2.jar:git-Paper-307]
[18.04 21:59:09] [Server] 	at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

**Comment by reneg1990** [comment]
If you would like LoneDev, I could record a video of this and also the issue where we cant leave the wardrobe if pressing Shift haha

**Comment by LoneDev6** [comment]
Also did you create any balloon?
Try to remove the new ballons you added. They probably miss the idle animation.

**Comment by LoneDev6** [comment]
This is not a log of when the issue occurred, I cannot find any errors about my plugins

**Comment by reneg1990** [comment]
[latest.log](https://github.com/PluginBugs/Issues-CosmeticsCore/files/11355081/latest.log)
I am so sorry about the LD! It appears i uploaded the wrong Latest log haha. I uploaded a new one just now. The errors related to cosmeticscore balloons is at the very end of the log

**Comment by LoneDev6** [comment]
Will be checked asap

**Comment by LoneDev6** [comment]
Based on the error I can tell you that ItemsAdder somehow fails to load the animation for the balloon.
Was the balloon edited?

Basically the ballon model seems not to contain any animation, so it fails to load.

**Comment by reneg1990** [comment]
Hey! Balloon was not edited. Fresh install from provided resource pack. I can't send any files currently as I am out on vacation till the 26th

________________________________
Fra: LoneDev ***@***.***>
Sendt: 9. maj 2023 12:38
Til: PluginBugs/Issues-CosmeticsCore ***@***.***>
Cc: reneg1990 ***@***.***>; Author ***@***.***>
Emne: Re: [PluginBugs/Issues-CosmeticsCore] Balloons not working - Issue on console related to LoneLibs/ItemsAdder? (Issue #127)


Based on the error I can tell you that ItemsAdder somehow fails to load the animation for the balloon.
Was the balloon edited?

Basically the ballon model seems not to contain any animation, so it fails to load.

**Comment by LoneDev6** [comment]
It's okay I wait for news as soon as you return, have a nice vacation
